src/renderers/WebGLRenderer.js

```diff
@@ -2801,7 +2801,7 @@ function WebGLRenderer( parameters ) {
 
 	};
 
-	this.copyTextureToTexture = function ( position, srcTexture, dstTexture, level ) {
+	this.copyTextureToTexture = function ( position, srcTexture, dstTexture, level = 0 ) {
```

Comment:
Perhaps we're inevitably headed toward using ES6, but for now there are no occurrences of default parameters in the library (that I know of?) and our Rollup build does not do a general-purpose ES6->ES5 transpile. Until an explicit decision is made to start using ES6, I think it'd be best to do an old-fashioned `level = level || 0` instead here.